lms/djangoapps/teams/static/teams/js/views/team_join.js

```diff
@@ -0,0 +1,132 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'text!teams/templates/team-join.underscore'],
+       function (Backbone, _, gettext, team_join_template) {
+           return Backbone.View.extend({
+
+               errorMessage: gettext("An error occurred. Please try again."),
```

Comment:
Thanks @andy-armstrong :-) 
We could simply say:  "An error occurred. Try again."
(Although I'm curious -- what circumstances do we use this error message in. Are they situations where our users might actually get a different result when they try again? If not, we should try to come up with different wording, such as: "An error occurred. Check (what?) and try again.")
Any insights/suggestions from you or @muhammad-ammar ?